services/contacts/acl.json

```diff
@@ -1,6 +1,35 @@
 {
-  "allowedEverybody": {
-    "allowedOrigins": [".*"],
-    "rules": {}
+  "Internal Tests": {
+    "allowedOrigins": [
+      "^https://.*\\.telefonica\\.com/WebAPI_pf.*",
+      "^https://.*\\.github\\.io/WebAPI_pf.*"
+    ],
+    "rules": {
+      "getAll": [{
+        "throttle": {
+          "maxPetitionsPerSecond": 1
+        },
+        "params": {
+          "sortBy": "(familyName)",
+          "sortOrder": "(ascending)",
+          "filterBy": "(familyName|firstName)",
+          "filterOp": "(startsWith|equals)",
+          "filterLimit": 50
+        }
+      }],
+      "save": [],
```

Comment:
Not sure if we should add rules for contacts. Per example, the name cannot be numeric or something like that.

WDYT?